tools/builder.js

```diff
@@ -95,9 +95,6 @@ var Builder = {
         name: "profile-w3c-common",
         logLevel: 2, // Show uglify warnings and errors.
         deps: [
-          "core/jquery-enhanced",
```

Comment:
These deps are now automatically included from profile-w3c-common.js (Require figures it out on its own).